CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## [Unreleased]
 
+## [1.5.0] - 2023-03-30
+
+### Configuration
+
+#### Bugfixes
+
+- Remove `SPLUNK_METRICS_ENDPOINT` from Instrumentation Libraries
+  (it was never really stable).
+
+#### Enhancements
+
+- Add `SPLUNK_PROFILER_MEMORY_ENABLED`.
+- Deprecate `jaeger-thrift-splunk` option for `OTEL_TRACES_EXPORTER`.
+- Remove the policy regarding Zipkin exporter.
```

Comment:
I'm not sure these are "enhancements". Maybe they would better fit under "breaking changes"?

Reply:
I am also not sure if it is a breaking change. The policy was that we should not document Zipkin exporter support.

From 1.4.0 tag:

https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/configuration.md?plain=1#L221-L222

Normally I would put it to "Removed"

Reply:
Agreed, I feel the allowed sections are lacking here. If you don't think there is a better place I'm fine resolving this comment.